go/zeusmaster/zeusmaster_test.go

```diff
@@ -0,0 +1,265 @@
+package zeusmaster_test
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"os"
+	"path"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/burke/zeus/go/config"
+	slog "github.com/burke/zeus/go/shinylog"
+	"github.com/burke/zeus/go/unixsocket"
+	"github.com/burke/zeus/go/zeusmaster"
+)
+
+var testFiles = map[string]string{
+	"zeus.json": `
```

Comment:
Yea, the trouble is that you can't access the directory of the particular test file in Go. So you can have an assets directory and just assume people are executing from the root or you can pull in a file embedding library, build in the strings and write them out just like this.